test/TestProjects/MgmtTest-TypeSpec/PrivateEndpointConnection.tsp

```diff
@@ -0,0 +1,43 @@
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+
+namespace Microsoft.MgmtTest;
+
+model PrivateEndpointConnectionResource
```

Comment:
Isn't one model and one operation enough?

Reply:
It is. Removed the other operations